src/components/CreateEvent/CreateEvent.tsx

```diff
@@ -90,23 +92,38 @@ export default function CreateEvent(props: CreateEventProps) {
     return state.appState.categoriesArray;
   });
 
+  const handlePictureChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files === null || e.target.files.length !== 1) {
+      return;
+    }
+    const id = `${v4()}${user.uid}`;
+
+    const file = e.target.files[0];
+
+    const storageRef = getStorage().child(id);
+    storageRef
+      .put(file)
+      .then(() => {
+        return storageRef.getDownloadURL();
+      })
+      .then((link) => {
+        setPhotoURL(link);
+      });
+  };
+
   return (
-    <Dialog
-      onClose={props.handleClose}
-      aria-labelledby="customized-dialog-title"
-      open={props.openCreateEvent}
-      maxWidth="md"
-      fullWidth={true}
-    >
+    <Dialog onClose={props.handleClose} open={props.openCreateEvent} maxWidth="md" fullWidth={true}>
       <Container maxWidth="lg">
         <Grid container={true} direction="column" justify="center" alignItems="center">
           <Grid item>
-            <DialogTitle id="customized-dialog-title">
-              <Typography variant="h6">Create Event</Typography>
+            <DialogTitle>
+              <Typography className={classes.title} variant="h6" component="span">
```

Comment:
Without span, the React throws errors

Reply:
I was wondering how to fix that error, thanks